ONSdigital/census-rm-case-processor
# isHouseholder Flag Added To Refusal Event

# Motivation and Context
Need to know if the respondent who refused is the householder along with their name and address details
isHouseholder flag is independent of Contact and Address blocks in event JSON so even if isHouseholder is false, Contact and Address may still have details in them

# What has changed
isHouseholder flag added to Refusal DTO (defaults to false if not supplied)
Contact and Address fields added to Refusal DTO
Report removed
Tests added

# How to test?
Maven Clean Install
Check tests pass and image builds
Run ATs with new image (although changes are minimal)
Try sending refusal event JSON on rabbit to check that isHouseholder is true when set and is false otherwise (especially when not included on event)

# Links
https://trello.com/c/WT0EGYyU/1150-nc-update-refusal-event-5
https://collaborate2.ons.gov.uk/confluence/display/CATD/Response+Management+Event+Dictionary?focusedCommentId=20210501#comment-20210501
